x/axelarnet/rate_limit.go

```diff
@@ -0,0 +1,50 @@
+package axelarnet
+
+import (
+	"github.com/axelarnetwork/axelar-core/x/axelarnet/keeper"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	capabilitytypes "github.com/cosmos/cosmos-sdk/x/capability/types"
+	porttypes "github.com/cosmos/ibc-go/v3/modules/core/05-port/types"
+	ibcexported "github.com/cosmos/ibc-go/v3/modules/core/exported"
+)
+
+type RateLimiter struct {
+	keeper  keeper.Keeper
+	channel porttypes.ICS4Wrapper
+	// accountKeeper  *authkeeper.AccountKeeper
+	// bankKeeper     *bankkeeper.BaseKeeper
+	// paramSpace     paramtypes.Subspace
+}
+
+func (i RateLimiter) SendPacket(ctx sdk.Context, chanCap *capabilitytypes.Capability, packet ibcexported.PacketI) error {
+	// contract := i.GetParams(ctx)
+	// if contract == "" {
+	// 	// The contract has not been configured. Continue as usual
+	// 	return i.channel.SendPacket(ctx, chanCap, packet)
+	// }
+
+	// amount, denom, err := GetFundsFromPacket(packet)
+	// if err != nil {
+	// 	return sdkerrors.Wrap(err, "Rate limited SendPacket")
+	// }
+	// channelValue := i.CalculateChannelValue(ctx, denom)
+	// err = CheckAndUpdateRateLimits(
+	// 	ctx,
+	// 	i.ContractKeeper,
+	// 	"send_packet",
+	// 	contract,
+	// 	channelValue,
+	// 	packet.GetSourceChannel(),
+	// 	denom,
+	// 	amount,
+	// )
+	// if err != nil {
+	// 	return sdkerrors.Wrap(err, "Rate limited SendPacket")
+	// }
```

Comment:
why commented out?

Reply:
the commits got a bit messed up for this. dropping this PR anyways in favour of using axelarnet deactivation as a way to disable IBC